pytorch_lightning/trainer/properties.py

```diff
@@ -181,7 +181,7 @@ def optimizers(self, new_optims: Optional[List[Optimizer]]) -> None:
         self.accelerator.optimizers = new_optims
 
     @property
-    def lr_schedulers(self) -> Optional[list]:
+    def lr_schedulers(self) -> List:
```

Comment:
Could you update the setter below to also ensure that it only accepts a List too?

Reply:
Also, this could be annotated as `List[Union[LRSchedulerTypeTuple]]`

with `from pytorch_lightning.utilities.types import LRSchedulerTypeTuple`

Reply:
> Also, this could be annotated as List[Union[LRSchedulerTypeTuple]]

this is now raising `mypy` errors. Do you think we could resolve these in a separate PR ?

Reply:
[union directly](https://github.com/PyTorchLightning/pytorch-lightning/pull/9419/commits/b82f802e531021b5d76f27b9d80d6b97d9142730) passes if that looks good?